src/APIBasedAuth.ts

```diff
@@ -197,11 +197,9 @@ class APIBasedAuth {
         '/client-tokens/redeem',
         { clientId: this.clientOptions.clientId, code }
       );
-    await this._store({
-      _type: 'token',
-      accessToken: data.accessToken,
+    await this.setTokens({
       idToken: data.identityToken,
-      refreshToken: data.refreshToken
+      ...data
```

Comment:
do we want to keep naming them here? not that it's a big deal but technically
```
{
  accessToken: '',
  identityToken: '',
  idToken: '',
  refreshToken: '',
}
```
are all passed through instead of just access/id/refresh